src/lib/components/Swap/Settings/MockToggle.tsx

```diff
@@ -1,6 +1,6 @@
 import { Trans } from '@lingui/macro'
 import { useAtom } from 'jotai'
-import { mockTogglableAtom } from 'lib/state/swap'
+import { mockTogglableAtom } from 'lib/state/settings'
```

Comment:
This file is only here so we don't have to recreate work on toggles later on. Thanks for keeping it up to date!